modules/building/pages/creating-secrets.adoc

```diff
@@ -86,14 +88,16 @@ type: kubernetes.io/dockerconfigjson
 
 [NOTE]
 ====
-* **Automatic Secret Linking via UI**
-  The {ProductName} UI automatically links image pull secrets to the `appstudio-pipeline` ServiceAccount.
+* **Secret Linking via UI**
+  The {ProductName} UI links image pull secrets to either selected or all components
+    specific service accounts `build-pipeline-$COMPONENT_NAME`
   - No pipeline YAML changes are required.
   - Tasks like the buildah task will use this secret automatically when pulling images from `quay.io`. (https://github.com/konflux-ci/build-definitions/blob/main/task/buildah-oci-ta/0.3/buildah-oci-ta.yaml in volumes)
 
 * **Manual Secret Creation**
   . **Link Secret To SA**
-    - If you create the secret manually (e.g., via `kubectl` or YAML), you must **manually link it** to the `appstudio-pipeline` ServiceAccount in your namespace.
+    - If you create the secret manually (e.g., via `kubectl` or YAML), you must **manually link it**
```

Comment:
I think we need to explain that secrets used in the task parameters should not be linked and refer to `Creating task input secrets`.